### ISSUE TITLE
Feature request: event topic status icon

### ISSUE THREAD

**Issue by almereyda** [angusmcleod/discourse-events#41]
The discourse-locations plugin offers a setting that is called

- location topic status icon
  Enable location topic status icons in all topic lists.

which shows a small map marker next to location-enabled topics in all topic lists.

In alignment of the integration between the two plugins here and there, it could be nice to also have a setting that enables an

- event topic status icon
  Enable event topic status icons in all topic lists.

as it is currently already shown in events-enabled categories to increase visibility of events.

**Comment by angusmcleod** [comment]
Yes, good suggestion on the setting, however you should actually currently see the event status icon on all lists if a topic has an event. See for example: https://discourse.angusmcleod.com.au/

<img width="542" alt="screen shot 2019-02-11 at 9 04 27 am" src="https://user-images.githubusercontent.com/5931623/52540362-c9034300-2ddc-11e9-8533-0f6c89987bcb.png">
